frame: "map"
agent_frame: "corrected_base_link"
dist_server_node_name: "dist_server"

sdg:
  min_jbubbles_rad: 0.4
  min_pbubbles_rad: 0.25
  bubbles_dist_offset: 0.0
  knn_checks: 50
  path_subdiv_length: 0.2
